client/app/components/CreateSourceDialog.jsx

```diff
@@ -116,6 +116,15 @@ class CreateSourceDialog extends React.Component {
           )}
         </div>
         <DynamicForm id="sourceForm" fields={fields} onSubmit={this.createSource} feedbackIcons hideSubmitButton />
+        {selectedType.type === "databricks" && (
+          <small>
+            By using the Databricks Data Source you agree to the Databricks JDBC/ODBC{" "}
+            <a href="https://databricks.com/spark/odbc-driver-download" target="_blank" rel="noopener noreferrer">
```

Comment:
I don't see the terms and conditions on this page.

![driver-download-page](https://user-images.githubusercontent.com/17067911/79741889-a2397a00-82c7-11ea-8629-9926bb2b34d3.png)

Reply:
I can see it. Maybe some adblocker?

Reply:
Strange. It's not an adblocker. Just not compatible in Firefox. Loads fine in Chrome and Safari.